content/news/playtest-20150830.md

```diff
@@ -0,0 +1,22 @@
+Another playtest on the road to our next stable release is ready!
+
+Being in a feature freeze, we focused on fixing bugs and inconsistencies reported by community members to ensure the release is as bug-free as possible.
+
+More notable things that got fixed include:
+
+* An exploit with spies in RA introduced with the new spy ability.
+* Issues with crushing infantry and walls.
+* Issues with entering transports.
+* Visual errors connected to shroud and fog of war.
+
+For details check out the [full changelog](http://changelog.openra.net).
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150830-new-ra-production-overlays.png)
+
+Production icons in RA got a new overlay that is more fitting of the RA95 style.
```

Comment:
I'd word this as something like "The spy production veterancy overlay has been improved to better fit the RA icon style"